### PR TITLE
Release 0.3.5

### PR DESCRIPTION
**Pull request by 928PJY**

### CODE REVIEW

**Comment by 928PJY** [comment]
@928PJY  test

**Comment by 928PJY** [comment]
@928PJY test

File: README.md [hunk 1]
```diff
@@ -162,6 +162,11 @@ Please try the following solutions:
 
 [MIT](https://raw.githubusercontent.com/docascode/vscode-docs-build/master/LICENSE)
 
+## Privacy statements
+
+For Microsoft employee privacy: https://privacy.microsoft.com/en-US/data-privacy-notice
```
Comment (herohua) [inline README.md]: just `For Microsoft employee`

**Review by herohua — APPROVED** [review]

**Pull request merged**